package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "graphql-query-generator",
-  "version": "0.5.5",
+  "name": "@brysgo/graphql-query-generator",
```

Comment:
Just make sure the `@brysgo/` goes away along with the different version number if this gets merged.